ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -23,16 +23,29 @@ declare LEASE_NUMBER
 declare VCENTER_AUTH_PATH
 
 # For leases >= 221, run on the IBM Cloud vSphere env
-if [ ${LEASE_NUMBER} -ge 221 ]; then
-  echo Scheduling job on IBM Cloud instance
-  VCENTER_AUTH_PATH=/var/run/vault/devqe-secrets/secrets.sh
-  vsphere_url="vcenter.devqe.ibmc.devcluster.openshift.com"
-  vsphere_datacenter="DEVQEdatacenter"
-  cloud_where_run="IBMC-DEVQE"
-  dns_server="192.168.${LEASE_NUMBER}.1"
-  vsphere_resource_pool="/DEVQEdatacenter/host/DEVQEcluster/Resources/ipi-ci-clusters"
-  vsphere_cluster="DEVQEcluster"
-  vsphere_datastore="vsanDatastore"
+#if [ ${LEASE_NUMBER} -ge 221 ]; then
```

Comment:
@sgaoshang will will be moved to vault like the other vsphere environments.

https://github.com/jcpowermac/ibm-vlan-subnet-automation/blob/main/scripts/ibm.sh

```
cat ibm.sh | vault kv patch -address https://vault.ci.openshift.org kv/selfservice/vsphere-vmc/config ibm.sh=-
```